price/price.go

```diff
@@ -17,6 +17,13 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const ETHGWeiCommaShift = 9
+const GWeiWeiCommaShift = 9
+const ETHWeiCommaShift = ETHGWeiCommaShift + GWeiWeiCommaShift
+const ETHinGWei int64 = 1000000000
+const GWeiInWei int64 = 1000000000
+const ETHinWei = ETHinGWei * GWeiInWei
```

Comment:
**minor:** Aside from `GWeiInWei` these aren't used.
I would delete them and instead of `GWeiInWei` use "10^`GWeiWeiCommaShift`" where it is used.

Reply:
removed.